buffalo/cmd/root.go

```diff
@@ -44,7 +44,6 @@ var RootCmd = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
-		fmt.Println(err)
```

Comment:
Instead of the following

```
buffalo new coke --db-type SOMETHING-BAD!
Buffalo version 0.7.1

Error: Unknown db-type SOMETHING-BAD! expecting one of postgres, mysql or sqlite3
Usage:
  buffalo new [name] [flags]

Flags:
      --db-type string   specify the type of database you want to use [postgres, mysql, sqlite3] (default "postgres")
  -f, --force            delete and remake if the app already exists
      --skip-pop         skips adding pop/soda to your app
      --skip-webpack     skips adding Webpack to your app
  -v, --verbose          verbosely print out the go get/install commands```

You would get the error out put again at the end. Shown below.

```
buffalo new coke --db-type SOMETHING-BAD!
Buffalo version 0.7.1

Error: Unknown db-type SOMETHING-BAD! expecting one of postgres, mysql or sqlite3
Usage:
  buffalo new [name] [flags]

Flags:
      --db-type string   specify the type of database you want to use [postgres, mysql, sqlite3] (default "postgres")
  -f, --force            delete and remake if the app already exists
      --skip-pop         skips adding pop/soda to your app
      --skip-webpack     skips adding Webpack to your app
  -v, --verbose          verbosely print out the go get/install commands

Unknown db-type SOMETHING-BAD! expecting one of postgres, mysql or sqlite3```